src/components/MemeberCard/MemberCard.js

```diff
@@ -0,0 +1,44 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Badge } from 'reactstrap';
+import Button from '../Button';
+import './memberCard.scss';
+
+const MemberCard = ({ certified }) => (
+  <div className="memberCard">
+    <h5>
+      {certified
+        ? (
+          <Fragment>
+            <span className="badge">
+              <Badge href="#" color="success">Certified</Badge>
+            </span>
+            <span className="expTime">9 months </span>
+            till your certification expires, click arrow to retake test
+            <Link to="/dashboard">
+              <FontAwesomeIcon icon="arrow-right" className="arrowIcon" />
+            </Link>
+          </Fragment>
+        )
+        : (
+          <Fragment>
+            You have not enrolled in any course
+            <Link to="/dashboard">
```

Comment:
Should I be redirected to the dashboard since I'm being asked to retake the test? 🤔